app/Cardano/Shell/Features/Logging.hs

```diff
@@ -27,8 +27,7 @@ import           Cardano.Shell.Types (CardanoConfiguration, CardanoEnvironment,
 --------------------------------
 
 data LoggingParameters = LoggingParameters
-    { configFp :: FilePath
-    , prefixFp :: FilePath
+    { lpFilePath :: !FilePath
```

Comment:
The idea is that here you have actual parameters from the logging module, and when you load them in the initialization, you pass them on.